select2/models/descriptors.py

```diff
@@ -1,7 +1,10 @@
 import django
 from django.db import router
 from django.db.models import signals
-from django.db.models.fields.related_descriptors import ReverseManyToOneDescriptor
+try:
+    from django.db.models.fields.related import ReverseManyRelatedObjectsDescriptor as ReverseManyToOneDescriptor
+except ImportError:
+    from django.db.models.fields.related_descriptors import ReverseManyToOneDescriptor
```

Comment:
Could you change this to `django.db.models.fields.related`? That's the convenience import path that [django itself uses](https://github.com/django/django/blob/6729b96d8a15048b2295c916c5b881a59d9417a0/django/contrib/contenttypes/fields.py#L11-14)